repose-aggregator/src/docs/asciidoc/recipes/index.adoc

```diff
@@ -3,7 +3,7 @@
 include::../_includes/in-progress.adoc[]
 
 == <<quick-start.adoc#,Quick Start>>
-If you want to stand up a *Repose* instance quickly, visit the Quick Start.
+If you want to stand up a *Repose* instance quickly.
```

Comment:
Since this is the same page as the one linked to from the primary index, I would just use the same description.
> Learn how to stand up *Repose* as quickly as possible!